### PR TITLE
Register new contexts, fix ed25519 context links.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
Rubberstamping ... didn't double check the exact registry values.

File: index.html [hunk 1]
```diff
@@ -651,12 +651,12 @@ <h1>Term Codec Registry</h1>
         </tr>
         <tr>
           <td><code>0x13</code></td>
-          <td><code>https://w3id.org/security/ed25519-signature-2018/v1</code></td>
+          <td><code>https://w3id.org/security/ed25519-2018/v1</code></td>
```
Comment (msporny) [inline index.html:654]:
```suggestion
          <td><code>https://w3id.org/security/suites/ed25519-2018/v1</code></td>
```

File: index.html [hunk 2]
```diff
@@ -651,12 +651,12 @@ <h1>Term Codec Registry</h1>
...
           <td>Ed25519Signature2018 Suite</td>
         </tr>
         <tr>
           <td><code>0x14</code></td>
-          <td><code>https://w3id.org/security/ed25519-signature-2020/v1</code></td>
+          <td><code>https://w3id.org/security/ed25519-2020/v1</code></td>
```
Comment (msporny) [inline index.html:659]:
```suggestion
          <td><code>https://w3id.org/security/suites/ed25519-2020/v1</code></td>
```

**Review by msporny — CHANGES_REQUESTED** [review]
Cross-checking against https://github.com/perma-id/w3id.org/blob/master/security/.htaccess#L34-L39 shows some discrepancies.

**Pull request merged**